### PR TITLE
Mobile Dashboard Banner Added

### LINKED ISSUE #796: Add Promotion Banner
Add Promotion Banner as shown in Website to dashboard activity
Screenshot 2020-05-18 at 1 42 50 PM

### PR DESCRIPTION
**Pull request by prathamk22**
Fixes #796 

Changes: Added dashboard banner api call to get the banner and showing it in the dashboard home fragment

Screenshots for the change:

WhatsApp Image 2020-08-06 at 01 00 53

### CODE REVIEW

**Comment by prathamk22** [comment]
@aggarwalpulkit596 pls review this.

**Comment by aggarwalpulkit596** [comment]
Can you place this on explore page And also wrap it around h a card to make it look similar to other ui

**Comment by prathamk22** [comment]
@aggarwalpulkit596 Done
Though the banner is empty as sent by the api. So its just to show you the container. Otherwise it will be gone.
WhatsApp Image 2020-08-07 at 14 50 22

**Comment by aggarwalpulkit596** [comment]
Can you show it using a sample image just to get an idea how it would look like

**Comment by prathamk22** [comment]
@aggarwalpulkit596 Bhaiya spin-win ka lagaya hai.Essa dikh rha hai

WhatsApp Image 2020-08-08 at 15 32 17

**Comment by prathamk22** [comment]
@aggarwalpulkit596 ?

**Comment by aggarwalpulkit596** [comment]
Can you do one thing
Move this container to below tracks recycler view and that would be good enough i guess

**Review by aggarwalpulkit596 — APPROVED** [review]

File: app/src/main/java/com/codingblocks/cbonlineapp/dashboard/explore/DashboardExploreFragment.kt [hunk 1]
```diff
@@ -143,6 +148,22 @@ class DashboardExploreFragment : BaseCBFragment() {
             swipeToRefresh.showSnackbar(it, anchorView = activity?.dashboardBottomNav, action = false)
         }
 
+        vm.fetchBanner().observer(viewLifecycleOwner){
+            bannerHolder.isVisible = it?.size?:0 > 0
+            if (it?.size?:0 > 0){
+                val bannerItem = it?.get(0)
+                bannerUrl = bannerItem?.link?:""
+                Glide.with(requireContext()).load(bannerItem?.mobileImageUrl).into(banner)
+            }
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/dashboard/explore/DashboardExploreFragment.kt:158]: Also use the loadimage extension function

**Pull request merged**